lib/samples/manage_features/manage_features.dart

```diff
@@ -0,0 +1,433 @@
+              ),
+              value: operation,
+              child: Text(getLabel(operation)),
+            ),
+          ),
+        )
+        .toList();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Container(
+                      padding: const EdgeInsets.all(5),
+                      child: Column(
+                        children: [
+                          // Create a dropdown button to select a feature management operation.
+                          DropdownButton(
+                            alignment: Alignment.center,
+                            hint: const Text(
+                              'Select operation',
+                              style: TextStyle(color: Colors.deepPurple),
+                            ),
+                            value: _selectedOperation,
+                            icon: const Icon(
+                              Icons.arrow_drop_down,
+                              color: Colors.deepPurple,
+                            ),
+                            elevation: 16,
+                            style: const TextStyle(color: Colors.deepPurple),
+                            // Set the onChanged callback to update the selected operation.
+                            onChanged: (operation) =>
+                                setState(() => _selectedOperation = operation),
+                            items: _featureManagementOptions,
+                          ),
+                          // Display additional UI depending on the selected operation.
+                          buildOperationSpecificWidget(),
+                        ],
+                      ),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  void onMapViewReady() async {
+    // Create and load a service geodatabase from a service URL.
+    const featureServiceURL =
```

Comment:
I think elsewhere we'd name URL values with "Uri" at the end, so `featureServiceUri`